### PR TITLE
Improve-order-flow: Add Fluffy 

### PR DESCRIPTION
**Pull request by ahdamin**
### Added

- Fluffy case/sample classes

### CODE REVIEW

File: cg/services/order_validation_service/errors/sample_errors.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
     MINIMUM_VOLUME,
 )
 from cg.services.order_validation_service.errors.order_errors import OrderError
-from cg.models.orders.sample_base import ContainerEnum
```
Comment (islean) [inline cg/services/order_validation_service/errors/sample_errors.py:6]: Was this not used?

File: cg/services/order_validation_service/workflows/fluffy/constants.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from cg.constants import DataDelivery
+
+
+class fluffyDeliveryType(Enum):
+    SCOUT = DataDelivery.SCOUT
+    NO_DELIVERY = DataDelivery.NO_DELIVERY
+
+
+class fluffyIndexEnum(Enum):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/fluffy/constants.py:11]:
```suggestion
class FluffyIndexEnum(Enum):
```

File: cg/services/order_validation_service/workflows/fluffy/models/sample.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from pydantic import Field
+
+from cg.models.orders.sample_base import ControlEnum, PriorityEnum
+from cg.services.order_validation_service.models.sample import Sample
+from cg.services.order_validation_service.workflows.fluffy.constants import (
+    fluffyIndexEnum,
+)
+
+
+class fluffySample(Sample):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/fluffy/models/sample.py:10]:
```suggestion
class FluffySample(Sample):
```

**Review by islean — APPROVED** [review]
Looks good although I would switch to the pythonic snake case and pascal case for namings. We should also double check how this works with https://github.com/Clinical-Genomics/improve-order-flow/issues/98

**Review by diitaz93 — APPROVED** [review]
Great 💯

**Comment by ahdamin** [comment]
Thanks @diitaz93

**Pull request merged**